### PR TITLE
Improve website UX and add restrictions on input files count and size

### LINKED ISSUE #35: Improve website UX and add restrictions on input files count and size
Add:
 - Favicon
 - Checks for website input files:
    - Maximum size
    - Number of input files
 - Badge to indicate the number of files sanitized.

Changed layout of website to house all the input/output controls in a nav bar.

### PR DESCRIPTION
**Pull request by the-c0d3br34k3r**
## Description
Closes #35.

Added:
  - Favicon
  - Checks for website input files:
     - Maximum size
     - Number of input files

Changed layout of website to house all the input/output controls in a nav bar.

## Author's Checklist
- [x] **These changes don't break existing dependants.** If they do, and you suggest to proceed, 
please explain why.

## Reviewer's Checklist
- [ ] **All classes have documentation comments.** - NA
- [ ] **All methods have documentation comments.** - NA
- [x] **No parameters are hardcoded.** Should be loaded as properties instead. If there needs to be hardcoded properties, explain why.
- [ ] **All information to be logged/displayed to the user are internationalized.** If not, explain why. - Will be addressed in #21.
- [x] **README.md has been updated if needed.**
- [ ] **pom.xml version is set to the latest date** If not, explain why. - NA

## Assignee's Checklist
- [ ] **All GitHub action checks have passed.**
- [ ] **All reviewers have approved.**
- [ ] **Relevant documentation has been updated if needed.**
- [ ] **pom.xml version is set to the latest date** If not, explain why.

### CODE REVIEW

File: script/lib/bootstrap.bundle.js
Comment (the-c0d3br34k3r) [inline script/lib/bootstrap.bundle.js:1]: Same version of bootstrap, just switched to using the whole bundle instead on the min bundle for certain additional functionality such as badges etc.

**Review by silvs110 — CHANGES_REQUESTED** [review]
When uploading an invalid har (aka one without any changes), it's not displayed in the unchanged files section. Also, it'll be good to disable the download button if the value of downloaded files is 0.
image

**Comment by silvs110** [comment]
Another thing to add.  The page gets stuck loading when uploading two files with the same name

File: README.md [hunk 1]
```diff
@@ -4,14 +4,20 @@
...
 The following browsers are supported:
  - Google Chrome
  - Firefox
 
-At the moment, it sanitizes HAR files.<br>
-Multiple files can be sanitized at the same time.<br>
-Support for sanitizing SAML requests/responses are in progress.
+### Limitations
+ - Maximum of 10 files can be sanitized at a time.
+ - Each file cannot exceed 50 MB.
```
[the remark this answers is not in the document]
Comment (the-c0d3br34k3r) [reply README.md:19]: Each file cannot exceed 50 MB. At the moment, a max of 10 files can be sanitized at a time.

**Comment by the-c0d3br34k3r** [comment]
> Another thing to add. The page gets stuck loading when uploading two files with the same name

How are you loading 2 files with the same name? Can you give an example?

**Comment by the-c0d3br34k3r** [comment]
> When uploading an invalid har (aka one without any changes), it's not displayed in the unchanged files section. Also, it'll be good to disable the download button if the value of downloaded files is 0. image

Do you have the latest changes in this branch? I'm able to see it in the unchanged files section and the download button is disabled.
Screenshot 2024-08-15 at 11 09 40 PM

**Comment by silvs110** [comment]
> > Another thing to add. The page gets stuck loading when uploading two files with the same name
> 
> How are you loading 2 files with the same name? Can you give an example?
Have file.har in directory A and another file.har (different content) in directory B

**Comment by silvs110** [comment]
> > When uploading an invalid har (aka one without any changes), it's not displayed in the unchanged files section. Also, it'll be good to disable the download button if the value of downloaded files is 0. image
> 
> Do you have the latest changes in this branch? I'm able to see it in the unchanged files section and the download button is disabled. Screenshot 2024-08-15 at 11 09 40 PM

My bad. The rebased version disables the downloaded button if there's no files to download

**Comment by the-c0d3br34k3r** [comment]
@silvs110, With the latest changes, when an invalid HAR file is selected, it shows an alert and reverts to the initial layout.
Screenshot 2024-08-19 at 1 16 36 PM

**Review by silvs110 — APPROVED** [review]

**Pull request merged**